CMakeLists.txt

```diff
@@ -489,7 +489,7 @@ add_module(gui/internal "Default" TRUE)
 add_module(map/binfile "Default" TRUE)
 add_module(map/filter "Default" TRUE)
 if(NOT MSVC)
```

Comment:
Why do we disable it also for Windows?

Reply:
See the main discussion. All others are fixed